include/ensmallen_bits/ens_version.hpp

```diff
@@ -21,7 +21,11 @@
 // (i.e. the version name will be "RC1", "RC2", etc.).  Otherwise the version
 // name will typically be a seemingly arbitrary set of words that does not
 // contain the capitalized string "RC".
-#define ENS_VERSION_NAME "No Direction Home"
+#define ENS_VERSION_NAME  "No Direction Home"
+// Incorporate the date the version was released
```

Comment:
```suggestion
// Incorporate the date the version was released.
```

Pedantic style issue, just to be consistent.